docs/oid-info.md

```diff
@@ -63,6 +77,24 @@ defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#secti
 Not used by Fulcio. This specifies the policy OID for the [timestamp authority](https://github.com/sigstore/timestamp-authority)
 that Sigstore operates.
 
-<!-- References -->
+## Mapping OIDC token claims to Fulcio OIDs
+
+| GitHub [(docs)][github-oidc-doc] | GitLab | CircleCI | Buildkite | Fulcio Certificate Extension | Why / Notes / Questions                                                                                                                                                                |
+|--------------------|--------|----------|-----------|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aud                | ??     | ??       | ??        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| iss                | ??     | ??       | ??        | Issuer                       | This already exists. For example: https://token.actions.githubusercontent.com                                                                                                          |
+| exp                | ??     | ??       | ??        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| nbf                | ??     | ??       | ??        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| iat                | ??     | ??       | ??        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| repository         | ??     | ??       | ??        | Source URI                   | Should include a fully qualified repository URL.                                                                                                                                       |
```

Comment:
I think right now GitHub's token has repository name here. I wonder of for backward compatibility it's worth to use `repository_url`?

Reply:
I don't think this claim exists atm for Actions. This probably means we would need to construct the source/repo URI in Fulcio based on the token issuer. Something like this for GHA:

```js
if (token.iss === 'https://token.actions.githubusercontent.com') {
  source_uri = `https://github.com/${token.repository}`
}
```